tests/functional/trace_latency_histogram.py

```diff
@@ -0,0 +1,142 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import time
+import pytest
+
+from api.iotrace_lat_hist_parser import LatencyHistogram
+from core.test_run import TestRun
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine, FioOutput
+from test_utils.size import Size, Unit
+from test_utils.os_utils import Udev
+
+
+@pytest.mark.parametrize("io_dir", [ReadWrite.write, ReadWrite.read])
+def test_latency_histogram_basic(io_dir):
+    """
+        title: Test for basic latency histogram properties
+        description: |
+            Test if samples count reported by fio equals count from iotracer (taking
+            into consideration dropped ones).
+        pass_criteria:
+            - Fio's samples number equals number of iotracer samples + number of
+              dropped ones
+    """
+    iotrace = TestRun.plugins['iotrace']
+    read = ReadWrite.read is io_dir
+
+    for disk in TestRun.dut.disks:
+        with TestRun.step(f"Start tracing on {disk.system_path}"):
+            iotrace.start_tracing([disk.system_path])
+            time.sleep(3)
+
+        with TestRun.step(f"Run {io_dir} IO"):
+            fio = (Fio().create_command(output_type=FioOutput.jsonplus)
+                   .io_engine(IoEngine.libaio)
+                   .size(Size(1, Unit.MebiByte))
+                   .block_size(Size(1, Unit.Blocks512))
+                   .read_write(io_dir)
+                   .target(disk.system_path)
+                   .direct())
```

Comment:
You could extract this to function with size parameter, because You use fio command more than once, nearly the same. It's only suggestion.

Reply:
I'll just leave as it is for now, hope that's ok :)